facetapi.block.inc

```diff
@@ -93,8 +93,7 @@ function facetapi_get_block_cache_setting($searcher, $realm_name) {
   if ('block' != $realm_name) {
     $name .= ':' . $realm_name;
   }
-  // TODO This variable was probably removed in Backdrop without replacement.
-  return variable_get($name, BACKDROP_NO_CACHE);
+  return facetapi_config($name, BACKDROP_NO_CACHE);
```

Comment:
Block cache was removed in Backdrop actually, so this could all be simplified

Reply:
https://api.backdropcms.org/api/backdrop/core%21includes%21common.inc/group/block_caching/1
I'm probably doing it wrong?